### PR TITLE
[testdriver.js] Document and test `bless` method

### PR DESCRIPTION
**Pull request by jugglinmike**
This new API was first introduced via [1]. Complete that initial work by
adding documentation and automated tests.

[1] 8b8082f30267410e4b3729439be23df8b803dde8

### CODE REVIEW

**Comment by jugglinmike** [comment]
[The method has been improved since I opened this pull request](https://github.com/web-platform-tests/wpt/pull/12399). Testing the improvement required rebasing this branch, and because no one has reviewed this yet, I force-pushed the result.

File: docs/_writing-tests/testdriver.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ the global scope.
...
 
+### `test_driver.bless(intent, action)`
+#### `intent: a string describing the motivation for this invocation`
+#### `action: an optional function`
+
+This function simulates [activation][activation], allowing tests to
+perform privileged operations that require user interaction. For
+example, sandboxed iframes with the
+`allow-top-navigation-by-user-activation` may only navigate their
+parent's browsing context under these circumstances. The `intent`
```
Comment (marcoscaceres) [inline docs/_writing-tests/testdriver.md]: Having used this now, I still find myself having to look up what the right text is to include is for “intent”. What you be great is an example of an intent string, and how it ends up being concatenated (reviewing on my phone, so hard to look up what the final string is).

**Review by marcoscaceres — APPROVED** [review]
Small request. The rest looks great.

File: docs/_writing-tests/testdriver.md (hunk 1 above)
Comment (jugglinmike) [reply docs/_writing-tests/testdriver.md]: An example sounds like a good idea to me! Added in a fixup commit

**Review by marcoscaceres — APPROVED** [review]
Double approve! 💯

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**